packages/client/src/api/db/types.d.ts

```diff
@@ -1,13 +1,14 @@
-import { Message, MessageInfoType, Wargame } from '@serge/custom-types'
+import { Message, MessageInfoType, Wargame, ActivityLogsInterface } from '@serge/custom-types'
 import DbProvider from '.'
 
 export interface DbProviderInterface {
   changes: (listener: (doc: Message | Wargame) => void) => void
   destroy: () => void
   get: (query: string) => Promise<Wargame | Message | { status: number }>
-  put: (doc: Wargame | Message) => Promise<Wargame | Message>
+  put: (doc: Wargame | Message | ActivityLogsInterface) => Promise<Wargame | Message | ActivityLogsInterface>
   allDocs: () => Promise<Message[]>
```

Comment:
@lilitkarapetyan - I think we need a different `put` event signature here, that takes an array of ActivityLogsInterface messages.  Or, I guess we could declare a new type that represents an array of ActivityLogsInterface items.

Reply:
Aah, @lilitkarapetyan - there is a _knock-on_ change from the above.  

We will no longer have a tidy list of activity for the current wargame.  Instead the `player-logs` database will have one lone list of activities.  So, we _may_ need to do some server-side processing to filter the activity for the specified wargame, to only include the newest entry for each role.